### PR TITLE
test: K6 Stress Testing Tool

### PR DESCRIPTION
**Pull request by wzalazar**
[PR Process](https://github.com/poetapp/documentation/blob/master/process/pr-review.md#pr-process) - [PR Review Checklist](https://github.com/poetapp/documentation/blob/master/process/pr-review.md#pr-review-checklist)

### Release
 Semantic release is enabled for this repository. Make sure you follow the right commit message convention. 
 We're using semantic-release's default — [Angular Commit Message Conventions](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines).

## Description of Changes

Fix https://github.com/poetapp/node/issues/755

This PR introduces:

- Script Endpoint to create Claims
- Test example, create `works`
- Documentation

This is an approach with `k6` https://docs.k6.io/docs.  

**Why is so difficult to create `works`?**

The endpoint works need a valid and unique claim for that, you must to use the library `poet-js`. For this reason, you need to create the claim first for use it after.

**why you do not create the all claim before and save it a file?**

it a possibility, but the problem is when you want to test for example concurrency with interval times. In this cases, you don't know how many claims you needed. Another problem is how to knows what claim you have to send.

Other libraries:

I researched `Artillery` and `JMeter`, and with these libraries is more complicated

**Steps to run the script:**

- Install K6 https://docs.k6.io/docs/installation
- `npm run create-claim-api`
- in the folder `tests/stress` `k6 run --vus 10 --duration 30s script.js`

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -72,6 +72,17 @@ services:
       -rpcallowip=::/0
       -deprecatedrpc=signrawtransaction
 
+  k6:
+    build: ./tests/stress
+    depends_on:
+      - poet-node
+    links:
```
Comment (warrenv) [inline docker-compose.yml]: links are no longer required. If you remove the links array, you should be able to docker-compose exec into the k6 container and ping poet-node.

Comment (wzalazar) [reply docker-compose.yml]: awesome, I will test

**Comment by wzalazar** [comment]
I tested in local and against an external domain and both works fine

for an external domain, I used https://regtest.qa.poetnetwork.net/works the last work is mine :)
or you can check if there is a work with

```
claim: {
  name: "testing claim 1542142376190",
  author: "the tester"
}
```

File: tests/stress/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+FROM node:10.12.0
```
Comment (geoffturk) [inline tests/stress/Dockerfile]: Change to 10.13.0

Comment (wzalazar) [reply tests/stress/Dockerfile]: cool, thanks @geoffturk !!!!

Comment (warrenv) [reply tests/stress/Dockerfile]: This was already done on master yesterday.

**Comment by wzalazar** [comment]
@warrenv I applied the same refactoring I did on the Frost api

**Pull request merged**